### DISCUSSION TITLE
Help with usage of CURLOPT_SSLKEY or Upgrade pgsql-http to 1.6.1 on supabase

### DISCUSSION THREAD

**Discussion by leomeneguzzi** [supabase/supabase#30766] [Questions]
I'm using Supabase cloud, and I know that Supabase already supports `pgsql-http` extension.
But it's on version `1.6`, and I need the `1.6.1` version https://github.com/pramsey/pgsql-http/releases/tag/v1.6.1
I need the support to:
- CURLOPT_SSLKEY_BLOB
- CURLOPT_SSLCERT_BLOB

I want to get the cert/key from the vault and use it on the request, but the param `CURLOPT_SSLKEY` only accepts a file path to load it, the `_BLOB` supports already loaded cert/key

So I need help to:
- Use the `CURLOPT_SSLKEY` and `CURLOPT_SSLCERT` on Supabase (I couldn't figure out how)
- Or make a request to Supabase to upgrade the `pgsql-http` extension to `1.6.1` (I don't know where I can create an issue/upgrade request)

**Comment by GaryAustin1** [comment]
Looks like the latest they install is 1.6.0 https://github.com/supabase/postgres/blob/118094e303f27642030f9b60118c30cb2d632905/nix/ext/pgsql-http.nix#L4 so no upgrading will get a newer version.

You could request they upgrade in that repository but no guarantee if and when they would do that.  Some extension versions have compatibility issues and they seem to check them out quite a bit before releasing into production.
